src/components/icon/icon.stories.mdx

```diff
@@ -46,12 +46,14 @@ const MyComponent = () => (
 
 ### With tooltip
 The Icon supports rendering tooltips internally, just pass a string to `tooltipMessage` to enable this feature. Other props, 
-such as `tooltipPosition` and `tooltipVisible` are also surfaced.
+such as `tooltipPosition`, `tooltipVisible` , `tooltipBgColor` and `tooltipFontColor` are also surfaced.
 
 <Preview>
   <Story name="with tooltip">
```

Comment:
We can disable this story for Chromatic

Reply:
I think they're disabled in the Meta of this aren't they?

Reply:
So they are